indexer/bag_indexer/src/main/scala/uk/ac/wellcome/platform/archive/indexer/bags/services/FileSuffix.scala

```diff
@@ -2,7 +2,7 @@ package uk.ac.wellcome.platform.archive.indexer.bags.services
 
 object FileSuffix {
   def getSuffix(name: String): Option[String] = {
-    val splitName = name.split("\\.")
+    val splitName = name.split(".+\\.")
```

Comment:
What's this testing? Maybe a quick comment here, about what it's meant to exclude?